includes/amp-helper-functions.php

```diff
@@ -44,7 +44,9 @@ function is_amp_endpoint() {
 		_doing_it_wrong( __FUNCTION__, sprintf( esc_html__( "is_amp_endpoint() was called before the 'parse_query' hook was called. This function will always return 'false' before the 'parse_query' hook is called.", 'amp' ) ), '0.4.2' );
 	}
 
-	return false !== get_query_var( AMP_QUERY_VAR, false );
+	$is_amp_endpoint = false !== get_query_var( AMP_QUERY_VAR, false );
```

Comment:
Why not just getting the result of `get_query_var( AMP_QUERY_VAR, false );`?

Reply:
Since it can be either an empty string or ''1''. If a URL is requested as `/?amp` or `/?amp=1` then this should cause AMP to be enabled in both cases.